ui/components/MeshModelRegistry/MeshModelDetails.js

```diff
@@ -152,15 +155,34 @@ const ModelContents = withSuppressedErrorBoundary(({ modelDef }) => {
     duplicates: modelDef?.duplicates?.toString(),
     relationships: getCompRelValue().relationships.toString(),
   };
-
+  const handleExport = () => {
+    const a = document.createElement('a');
```

Comment:
@Jougan-0 Instead UI can simply make a GET call, and the server is already responding with a Content-Disposition header, so no need to to this I believe.